elements_imaging/imaging.py

```diff
@@ -54,22 +51,23 @@ def activate(imaging_schema_name, scan_schema_name=None, *, create_schema=True,
 
 # -------------- Functions required by the elements-imaging  --------------
 
-def get_caiman_dir(processing_task_key: dict) -> str:
```

Comment:
Should we remove `get_caiman_dir` and `get_suite2p_dir` from `workflow-imaging/workflow_imaging/paths.py` and `workflow-imaging/workflow_imaging/pipeline.py`?

Reply:
yes, same as above, PR for `workflow-imaging` coming soon